pages/HomePage.ts

```diff
@@ -37,6 +39,8 @@ export class HomePage {
         this.ccLink = page.getByRole(`link`, { name: `CC 4.0 BY-NC` });
         this.pngUrl = `/image`;
         this.ccUrl = `/licenses`;
+        this.informationTitles = page.locator(`#overview .row .col-sm`);
```

Comment:
Why it located here? All locator should be in fields

Reply:
Fixed